src/plugin.js

```diff
@@ -232,19 +232,27 @@ class PlaylistMenu extends Component {
       this.addClass('vjs-mouse');
     }
 
-    player.on(['loadstart', 'playlistchange', 'playlistsorted'], (event) => {
+    this.on(player, ['loadstart', 'playlistchange', 'playlistsorted'], (event) => {
```

Comment:
This means we properly stop listening to player events if the component is disposed.